fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/control/JobControlContextListener.java

```diff
@@ -30,6 +32,9 @@
     @Override
     public void contextInitialized(ServletContextEvent sce) {
         log.info("Starting up the Bulk Data Web App");
+
+        FHIRConfiguration.setConfigHome(System.getenv("FHIR_CONFIG_HOME"));
```

Comment:
I just double checked javadoc

``` 
Only throws if the name passed to getenv
    NullPointerException - if name is null**
```

Reply:
yeah, i checked it too....if the env var isn't set, it will pass null.  and our contract for setConfigHome treats that like "".